package kh.java.func;

import kh.java.exception.UserException1;

public class Calc {
	public void div(int num1, int num2) {
		if(num2==0) {
			throw new UserException1("0으로는 나눌 수 없습니다.");
		}else {
			System.out.println((num1/num2));
		}
	}
}
